core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/release/BitbucketcloudReleaser.java

```diff
@@ -0,0 +1,284 @@
+        public boolean isSkipTag() {
+            return BitbucketcloudReleaser.this.isSkipTag();
+        }
+
+        @Override
+        public boolean isSkipRelease() {
+            return BitbucketcloudReleaser.this.isSkipRelease();
+        }
+
+        @Override
+        public boolean isOverwrite() {
+            return BitbucketcloudReleaser.this.isOverwrite();
+        }
+
+        @Override
+        public Update getUpdate() {
+            return BitbucketcloudReleaser.this.getUpdate().asImmutable();
+        }
+
+        @Override
+        public String getApiEndpoint() {
+            return BitbucketcloudReleaser.this.getApiEndpoint();
+        }
+
+        @Override
+        public boolean isArtifacts() {
+            return BitbucketcloudReleaser.this.isArtifacts();
+        }
+
+        @Override
+        public boolean isFiles() {
+            return BitbucketcloudReleaser.this.isFiles();
+        }
+
+        @Override
+        public boolean isChecksums() {
+            return BitbucketcloudReleaser.this.isChecksums();
+        }
+
+        @Override
+        public boolean isCatalogs() {
+            return BitbucketcloudReleaser.this.isCatalogs();
+        }
+
+        @Override
+        public boolean isSignatures() {
+            return BitbucketcloudReleaser.this.isSignatures();
+        }
+
+        @Override
+        public Active getUploadAssets() {
+            return BitbucketcloudReleaser.this.getUploadAssets();
+        }
+
+        @Override
+        public boolean isPrerelease() {
+            return BitbucketcloudReleaser.this.isPrerelease();
+        }
+
+        @Override
+        public CommitAuthor getCommitAuthor() {
+            return BitbucketcloudReleaser.this.getCommitAuthor().asImmutable();
+        }
+
+        @Override
+        public Map<String, Object> asMap(boolean full) {
+            return unmodifiableMap(BitbucketcloudReleaser.this.asMap(full));
+        }
+
+        @Override
+        public boolean isEnabled() {
+            return BitbucketcloudReleaser.this.isEnabled();
+        }
+
+        @Override
+        public String getOwner() {
+            return BitbucketcloudReleaser.this.getOwner();
+        }
+
+        @Override
+        public Integer getConnectTimeout() {
+            return BitbucketcloudReleaser.this.getConnectTimeout();
+        }
+
+        @Override
+        public Integer getReadTimeout() {
+            return BitbucketcloudReleaser.this.getReadTimeout();
+        }
+    };
+
+    public BitbucketcloudReleaser() {
+        super(org.jreleaser.model.api.release.BitbucketcloudReleaser.TYPE, true);
+        setHost("bitbucket.org");
+        setRepoUrl("https://{{repoHost}}/{{repoOwner}}/{{repoName}}");
+        setRepoCloneUrl("https://{{repoHost}}/{{repoOwner}}/{{repoName}}.git");
+        setCommitUrl("https:://{{repoHost}}/{{repoOwner}}/{{repoName}}/commits");
+        setSrcUrl("https://{{repoHost}}/{{repoOwner}}/{{repoName}}/src/{{repoBranch}}");
```

Comment:
I think there are other places that expect a non-null value for issueTracker like `project.links` and the changelog generator. Will have to null proof them after merging.

Reply:
Did a quick check and this won't be a problem.